plugin/src/test/resources/archunit_ignore_patterns.txt

```diff
@@ -1 +1,6 @@
-// empty up to now, see https://www.archunit.org/userguide/html/000_Index.html#_ignoring_violations
\ No newline at end of file
+# see https://www.archunit.org/userguide/html/000_Index.html#_ignoring_violations
+
+# ParserConfiguration.doCheckConsoleLogScanningPermitted does not require @POST or a permission check.
+# As-implemented, it does not leak data, even if called illegally.
+.*ParserConfiguration\.doCheckConsoleLogScanningPermitted.*POST.*
+.*Permission.*ParserConfiguration\.doCheckConsoleLogScanningPermitted.*
```

Comment:
I tried to avoid having to add any ignores, but there's no reason why that specific doCheck method requires a POST - that's only needed when a doCheck method can leak information.